keyboards/lime/rev1/keyboard.json

```diff
@@ -42,6 +42,15 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": false,
+        "extrakey": false,
+        "swap_hands": true,
+        "encoder": true,
+        "encoder_right": true,
```

Comment:
`ENCODER_RIGHT_ENABLE` seems to only have a single reference in the whole repo, the one in `keyboards/lime/rev1/rules.mk`
```suggestion
```

Reply:
I meant to ask what I should do about that and forgot. In any case, I got my answer.